### DISCUSSION TITLE
Has Addendum 15.1 "Modulation Quality with 500 kHz LoRa Bandwidth" been addressed in RadioLib?

### DISCUSSION THREAD

**Discussion by MrSniffer** [jgromes/RadioLib#701] [Q&A] [answered]
Semtech wrote:

Some sensitivity degradation may be observed on any LoRa device, when receiving signals transmitted by the SX1261/2 with a LoRa BW of 500 kHz.

15.1.2 Workaround: Before any packet transmission, bit #2 at address 0x0889 shall be set to: 0 if the LoRa BW = 500 kHz 1 for any other LoRa BW 1 for any (G)FSK configuration.

This workaround didn't appear to be applied to the sx126x.cpp code, in which case I will submit an issue. Can anyone confirm either way, kindly?

**Comment by jgromes** [comment, marked as answer]
This fix is already implemented, see `SX126x::fixSensitivity()`

**Reply by MrSniffer** [reply]
Wonderful news. Excellent to learn that is taken care of!
